drivers/dryer/device.js

```diff
@@ -17,7 +17,7 @@ module.exports = class SmartThingsDeviceDryer extends SmartThingsDevice {
     {
       homeyCapabilityId: 'samsung_dryer_progress_remaining_time',
       smartThingsComponentId: 'main',
-      smartThingsCapabilityId: 'samsungce.washerOperatingState',
+      smartThingsCapabilityId: 'samsungce.dryerOperatingState',
```

Comment:
We have not recieved any reports that these capabilities are broken. Could you elaborate on what is broken and how this solution fixes it?

Reply:
This fixes so that it's the dryer operation and not washer operation.
I did not get any updates on my dryer when this was set to washer, and when I went through Samsungs api the correct setting to get updates is 'dryerOperatingState' for dryer and not 'washerOperatingState' as that is for washing machines.